github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -2224,6 +2230,8 @@ periodics:
         value: "true"
       - name: TARGET
         value: k8s-1.26-centos9-sig-operator
+      - name: KUBEVIRT_PSA
```

Comment:
This will not work. We need to split as we have for centos8 or don't run it. Same for the presubmits.

Reply:
True, we need to do a similar thing for network, since Istio is blocking.

Reply:
Regarding periodics, I wouldn't split those, since they just tell us what is working and what isn't.

Reply:
In the operator case I think there might be a cascading issue - read one test fails and it will affect others.